server/base/src/main/java/org/apache/accumulo/server/manager/recovery/HadoopLogCloser.java

```diff
@@ -67,8 +67,6 @@ public long close(AccumuloConfiguration conf, Configuration hadoopConf, VolumeMa
         ns.append(source).close();
         log.info("Recovered lease on {} using append", source);
       }
-    } else if (ns instanceof LocalFileSystem || ns instanceof RawLocalFileSystem) {
```

Comment:
Do you happen to know if the local filessystem impls will have the `CommonPathCapabilities.LEASE_RECOVERABLE` capability?

Reply:
It looks like only DistributedFileSystem and ViewDistributedFileSystem implement the LeaseRecoverable interface.